.github/workflows/go.yml

```diff
@@ -0,0 +1,45 @@
+name: Go
+on: [push]
+jobs:
+
+  build:
+    name: Build
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macOS-latest, ubuntu-latest]
+        goversion: [1.12, 1.13]
```

Comment:
Let's drop `1.12` here. The code uses error-wrapping with `fmt.Errorf`, and I don't really want to give it up for this package.